src/Symfony/Component/Console/Completion/Output/BashCompletionOutput.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Completion\Output;
+
+use Symfony\Component\Console\Completion\Completion;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Wouter de Jong <wouter@wouterj.nl>
+ */
+class BashCompletionOutput implements CompletionOutputInterface
+{
+    public function write(Completion $completion, OutputInterface $output): void
+    {
+        $options = [];
+        foreach ($completion->getOptionSuggestions() as $option) {
+            $options[] = '--'.$option->getName();
+            /*if ($shortcut = $option->getShortcut()) {
+                $options[] = '-'.$shortcut;
+            }*/
```

Comment:
```suggestion
```

Can probably be removed, I think it doesn't make sense to complete short option names.